docs/site/src/docs/advanced/raw-docker-baseimage.md

```diff
@@ -38,7 +38,16 @@ Edit the file 'daemon.json' at `/var/lib/sealer/data/overlay2/{layer-id-1}/etc/`
 
 #### Step 7: add network components
 
-Now the base image still need network components to create k8s clusters, so move the file "tigera-operator.yaml" and the file "custom-resources.yaml" from `/var/lib/sealer/data/overlay2/{layer-id-2}/etc/` to `/var/lib/sealer/data/overlay2/{layer-id-1}/etc/`.
+Now the base image still need network components to create k8s clusters, so move the file "tigera-operator.yaml" and the file "custom-resources.yaml" from `/var/lib/sealer/data/overlay2/{layer-id-2}/etc/` to `/var/lib/sealer/data/overlay2/{layer-id-1}/etc/`. Then you need modify some contents in those two files to make sure the pods they create will pull docker images from your private registry, which will make your k8s clusters still work well in offline situations. In this case, firstly add a map-key value in "custom-resources.yaml", the key is `spec.registry` and the value is `sea.hub:5000`, secondly modify all docker image names in "tigera-operator.yaml" from `<registry>/<repository>/<imageName>:<imageTag>` to `sea.hub:5000/<repository>/<imageName>:<imageTag>`.
```

Comment:
Why not from the raw docker base image then add network plugin.
```
FROM raw-docker
COPY calico .
CMD kubectl apply -f calico
```
If put calico in layer1, what if user want flannel?

Reply:
good suggestions